pkg/crc/machine/bundle/constants.go

```diff
@@ -6,11 +6,21 @@ import (
 
 var bundleLocations = map[string]bundlesDownloadInfo{
 	"amd64": {
-		"darwin": download.NewRemoteFile("", ""),
-		"linux": download.NewRemoteFile(
-			"https://storage.googleapis.com/crc-bundle-github-ci/4.9.5/crc_libvirt_4.9.5.crcbundle",
-			"603d87599f4c129683d3a8a11bcd41f240a529b2dd81075755eebf3d2b8cacc6",
-		),
-		"windows": download.NewRemoteFile("", ""),
+		"darwin": {
+			"openshift": download.NewRemoteFile("", ""),
+			"podman":    download.NewRemoteFile("", ""),
```

Comment:
Would it be better to not add these dummy files and make sure a good error is returned? At the moment this gives:
```
INFO Downloading crc_hyperkit_4.9.5.crcbundle     
Get "": unsupported protocol scheme ""
```